blogvault/wpcom-migration
# Add wp-env support

This PR adds [wp-env](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/) support for easy environment setup.

# Testing Instructions

* Ensure that Docker is running.
* Install wp-env using `npm -g i @wordpress/env`.
* Run the `wp-env start` command.
* Go to http://localhost:8888/wp-admin (user: `admin`, pass: `password`) and make sure the plugin is running.
